### ISSUE TITLE
Help us improve our WAVE accessibility score

### ISSUE THREAD

**Issue by mekarpeles** [internetarchive/openlibrary#4082]
https://wave.webaim.org/report#/openlibrary.org

### Evidence / Screenshot (if possible)

![wave webaim org_report](https://user-images.githubusercontent.com/978325/98904419-671eb400-246e-11eb-961a-2f90f8f3c526.png)

### Relevant url?

https://wave.webaim.org/report#/openlibrary.org

### Proposal & Constraints

On homepage:
* [ ] 168 broken ARIA references
* [ ] 3 empty buttons
* [ ] 2 missing form labels

### Stakeholders

@bfalling

**Comment by Yashs911** [comment]
@mekarpeles I would like to work on this issue

**Comment by Yashs911** [comment]
@bfalling I investigated a little and found out that all 168 broken ARIA reference is due to the generation of `aria-describedby="slick-slide817` on further investigation I found this [issue](https://github.com/kenwheeler/slick/issues/2020) and according to them we should remove `aria-describedby` generation.
https://github.com/kenwheeler/slick/issues/2020#issuecomment-223031944

**Comment by Yashs911** [comment]
Adding `aria-label="Search input"` and `aria-label="(what should I write here)"` should solve missing form label

![label](https://user-images.githubusercontent.com/64412143/99068614-d7464a80-25d2-11eb-8073-1303e844fb7b.JPG)

**Comment by Yashs911** [comment]
Regarding empty buttons, this is what I found
> The button issue is coming from the Colorbox code in the libraries folder. You can add an aria-label to each button in the jQuery.colorbox.js file but you'll need to make sure that change is reflected in the minified version as well or it won't take effect.

**Comment by Yashs911** [comment]
@mekarpeles New Errors
- [ ] 196 X Broken ARIA reference
- [ ] 11 Contrast Errors (Because of "Not in Library" buttons) (Changing background color from `#999999` to `#595959` solves the problem.)

**Comment by nynaalekhya** [comment]
@mekarpeles 
Hi,
I'm interested in working on this issue. Could you please assign me this issue.
